scripts/test.py

```diff
@@ -0,0 +1,97 @@
+import os
+import re
+import unittest
+
+def model(model_name):
+  return '--model=../model_zoo/historical/prototext/%s.prototext' % model_name
+
+def reader(reader_name):
+  return '--reader=../model_zoo/data_readers/%s.prototext' % reader_name
+
+def optimizer(optimizer_name):
+  return '--optimizer=../model_zoo/optimizers/%s.prototext' % optimizer_name
+
+def executable(cluster):
+  return '../build/%s/model_zoo/lbann' % cluster
+
+def run_lbann(model_name, reader_name, optimizer_name, output_file_name):
+  if (model_name == None or reader_name == None or optimizer_name == None or output_file_name == None):
+    raise Exception("Invalid number of arguments")
+  else:
+    os.system('hostname > %s' % output_file_name)
```

Comment:
Inefficient to write to file just to get this information? Would it be better to use a subprocess?